### PR TITLE
welcome page updates

### PR DESCRIPTION
**Pull request by zbtirrell**
See: https://moderntribe.atlassian.net/browse/TEC-3559

related: https://github.com/moderntribe/tribe-common/pull/1419

### CODE REVIEW

File: src/admin-views/admin-update-message.php [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 ?>
 
-<p class="tribe-welcome-message"><?php echo esc_html( sprintf( __( 'You are running Version %s and deserve a hug :-)', 'the-events-calendar' ), Tribe__Events__Main::VERSION ) ); ?></p>
+<p class="tribe-update-message"><?php echo esc_html( sprintf( __( 'You are running Version %s and deserve a hug :-)', 'the-events-calendar' ), Tribe__Events__Main::VERSION ) ); ?></p>
```
Comment (tr1b0t) [inline src/admin-views/admin-update-message.php:9]: :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 88ffc025177ffd8d07d1105b9ecc79cd</sub>

Comment (lucatume) [reply src/admin-views/admin-update-message.php:9]:
```suggestion
/* translators: the placeholder will contain the semantic version of The Events Calenda plugin, e.g. 1.2.3 */
<p class="tribe-update-message"><?php echo esc_html( sprintf( __( 'You are running Version %s and deserve a hug :-)', 'the-events-calendar' ), Tribe__Events__Main::VERSION ) ); ?></p>
```

**Comment by lucatume** [comment]
Rebased this on `release/B20.08` @zbtirrell: was the `master` on purpose?

**Comment by zbtirrell** [comment]
I wasn't sure what branch / sprint was current, so `master` was me being lazy - thanks for fixing that.

File: src/admin-views/admin-update-message.php [hunk 2]
```diff
@@ -6,7 +6,8 @@
 
 ?>
 
-<p class="tribe-welcome-message"><?php echo esc_html( sprintf( __( 'You are running Version %s and deserve a hug :-)', 'the-events-calendar' ), Tribe__Events__Main::VERSION ) ); ?></p>
+/* translators: the placeholder will contain the semantic version of The Events Calenda plugin, e.g. 1.2.3 */
+<p class="tribe-update-message"><?php echo esc_html( sprintf( __( 'You are running Version %s and deserve a hug :-)', 'the-events-calendar' ), Tribe__Events__Main::VERSION ) ); ?></p>
```
Comment (tr1b0t) [inline src/admin-views/admin-update-message.php:10]: :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 9a48dbe92857984b8bfac16f21468c92</sub>

**Pull request merged**